pydicom/sopclass_uids.py

```diff
@@ -0,0 +1,299 @@
+"""
+Storage SOP Class UIDs auto-generated by generate_storage_sopclass_uids.py
+"""
```

Comment:
I'm wondering if to be consistent with the different automatically generated dicts we should have this be named starting with an underscore? Eg `_sopclass_uids.py`

Another approach to make it less likely for the user (when pressing TAB to see what is available) to find these core files would be to add them to a submodule. This would mean putting these scripts into their own folder as an entire module that is automatically generated. For example, the structure would look like:

```
pydicom/
    source/
        __init__.py
        sopclass_uids.py
        dicom_dict.py
        private_dict.py
        uid_dict.py
```
and then in __init__.py we could import them with whatever name is liked, eg:

```
from .dicom_dict import (
    DicomDictionary as dicom_dict,
    RepeatersDictionary as repeater_dict
)
```
and then that would give us control to expose/not expose different components from the hugely generated files. Then use would look like:

```
from pydicom.source import dicom_dict
```
and we might also make it so that the names of the generator files are used to determine the names of the outcome files, if we want to (eventually) have a programmatic process (an executable) for generating the content. That is probably overkill, but would be pretty easy to do (naming them akin to one another).